messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -510,6 +537,32 @@ Builder maxClaimedSegments(int maxClaimedSegments) {
             return this;
         }
 
+        /**
+         * Sets the initial segment count used to create segments on start up.
+         *
+         * @param initialSegmentCount an {@code int} specifying the initial segment count used to create segments on
+         *                            start up
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder initialSegmentCount(int initialSegmentCount) {
+            this.initialSegmentCount = initialSegmentCount;
+            return this;
+        }
+
+        /**
+         * Specifies the {@link Function} used to generate the initial {@link TrackingToken}s.
```

Comment:
It would be nice to tell what the default `initialToken` is on the JavaDoc here! :-)